src/Stratis.StratisDnsD/Properties/launchSettings.json

```diff
@@ -1,11 +1,11 @@
 {
-  "profiles": {
-    "Stratis.StratisDnsD": {
-      "commandName": "Project"
-    },
-    "Stratis.StratisDnsD Test": {
-      "commandName": "Project",
-      "commandLineArgs": "-testnet -dnslistenport=5399"
+    "profiles": {
+        "Stratis.StratisDnsD": {
+            "commandName": "Project"
+        },
+        "Stratis.StratisDnsD Test": {
```

Comment:
Can you create a third launch settings that has the dns stuff in, but leave the testnet one to be just -testnet?